src/tests/unit/tests/common/components/__snapshots__/telemetry-permission-dialog.test.tsx.snap

```diff
@@ -1,57 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`TelemetryPermissionDialogTest render dialog 1`] = `
-<Dialog
-  dialogContentProps={
-    {
-      "showCloseButton": false,
-      "title": "We need your help",
-      "type": 0,
-    }
-  }
-  hidden={false}
-  modalProps={
-    {
-      "className": "telemetry-permission-dialog-modal",
-      "containerClassName": "insights-dialog-main-override telemetry-permission-dialog",
-      "isBlocking": true,
-    }
-  }
->
-  <div
-    className="telemetry-permission-details"
-  >
-    <TelemetryNotice
-      deps={
-        {
-          "LinkComponent": [Function],
-        }
-      }
-    />
-  </div>
-  <div
-    className="telemetry-checkbox-section"
-  >
-    <StyledCheckboxBase
-      checked={true}
-      className="telemetry-agree-checkbox"
-      label="I agree to enable telemetry"
-      onChange={[Function]}
-    />
-    <PrivacyStatementPopupText
-      deps={
-        {
-          "LinkComponent": [Function],
-        }
-      }
-    />
-  </div>
-  <StyledDialogFooterBase>
-    <CustomizedPrimaryButton
-      className="start-using-product-button"
-      onClick={[Function]}
-      text="OK"
-    />
-  </StyledDialogFooterBase>
-</Dialog>
+<DocumentFragment>
```

Comment:
This snapshot is missing a ton of information. We should try to get the snapshots as close to the original as possible. I think the best way to do that for this one is to mock the `Dialog` component and call `expectMockedComponentPropsToMatchSnapshots([Dialog])`

Reply:
We cannot mock Dialog as we need it to test the checkbox and button inside the dialog but we have made the change to add missing dialog information in snapshot file.